corona-warn-app/cwa-app-android
# FIX: Fix time issue (DEV)

### Testing
- install CWA 2.22 set env. to mock-cloud
- App Information > Error Reports > Start > Send Error Report
- install CWA from this PR
- App Information > Error Reports > Start > Send Error Report
- ID History should contain previous and new error reports and app shouldn't crash
